src/IVSPlayer.tsx

```diff
@@ -239,7 +231,7 @@ const IVSPlayerContainer = React.forwardRef<IVSPlayerRef, Props>(
         duration: number | null;
       }>
     ) => {
-      if (!paused) {
+      if (autoplay) {
```

Comment:
```suggestion
      if (!paused || autoplay) {
```
If we're not paused or autoplay is enabled we should play